grammars/breakpoints.cson

```diff
@@ -0,0 +1,9 @@
+'scopeName': 'hydrogen.breakpoints'
+'name': 'Hydrogen breakpoints'
+'injectionSelector': 'comment'
```

Comment:
I think this should be 'comment.line'. Otherwise `/* %% */` would get highlighted in JS.